src/components/KMethodBadge/KMethodBadge.vue

```diff
@@ -0,0 +1,167 @@
+<template>
+  <KBadge
+    :background-color="backgroundColor || methodBadgeColors[method].backgroundColor"
+    :border-color="borderColor || methodBadgeColors[method].borderColor"
+    class="k-method-badge"
+    :class="[ `method-${method}`, isToggle ? 'k-method-toggle' : '']"
+    :color="color || methodBadgeColors[method].color"
+    :is-bordered="isToggle"
+    max-width="auto"
+    :shape="shape === 'rectangular' ? 'rectangular' : 'rounded'"
```

Comment:
```suggestion
    :shape="shape"
```
question: Wouldn't this work?

Reply:
Fixed